docs/source/quickstart/index.rst

```diff
@@ -223,7 +237,8 @@ schema.json file.
 Tab Completion
 --------------
 
-To enable command-line tab completion for `custodian` on bash do the following one-time steps:
+To enable command-line tab completion for `custodian` on bash do the following
+one-time steps:
```

Comment:
`do the following one-time steps` --> `complete the following steps. Note that you only need to do this once.`